### PR TITLE
Added B2B requirements for tenant to valid issuers

### PR DESCRIPTION
**Pull request by dke-benjy**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/double-key-encryption.md [hunk 1]
```diff
@@ -240,6 +240,7 @@ DKE tenant and key settings are located in the **appsettings.json** file.
      "https://sts.windows.net/9c99431e-b513-44be-a7d9-e7b500002d4b/"
    ]
    ```
+Please note: if you want to enable external B2B access to your key store, you will also need to include these external tenants as part of the valid issuers list.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/double-key-encryption.md]:
```suggestion
> [!NOTE]
> If you want to enable external B2B access to your key store, you will also need to include these external tenants as part of the valid issuers' list.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@dke-benjy

**Pull request merged**